src/main/controllers/DescribeWhatHappenedController.ts

```diff
@@ -14,13 +14,17 @@ export default class DescribeWhatHappenedController {
     fields: {
       claimSummaryText: {
         id: 'claim-summary-text',
+        label: 'Claim summary text',
+        labelHidden: true,
```

Comment:
Would it not be better for text changes like this to be added to the ,json files for each controller?

Reply:
Added for those that have existing json files.